token/cli/src/output.rs

```diff
@@ -176,6 +187,9 @@ impl fmt::Display for CliMultisig {
 #[serde(rename_all = "camelCase")]
 pub(crate) struct CliTokenAccount {
     pub(crate) address: String,
+    pub(crate) program_id: String,
```

Comment:
As another note, these are a breaking changes on the CLI output, which may cause issues for people who use json or json-compact and parse the data with another program.

We'll have to be careful in the release notes to specify this change